Unable to install on Windows ...

Ruby rookie here trying to install this native gem on Win10 x64.  Obtained libexif from here: 
  http://www.ridgesolutions.ie/index.php/2015/03/13/libexif-binaries-for-windows-and-visual-studio/libexif-win32-0-6-21/
and extracted into root & devkit folders.

Unable to get past============================
C:\Users\kart\GitHub\ruby225 (master)
λ gem install exif --verbose
HEAD https://api.rubygems.org/api/v1/dependencies
200 OK
GET https://api.rubygems.org/api/v1/dependencies?gems=exif
200 OK
Temporarily enhancing PATH to include DevKit...
C:/Users/kart/GitHub/ruby225/lib/ruby/gems/2.2.0/gems/exif-1.0.1/.gitignore
C:/Users/kart/GitHub/ruby225/lib/ruby/gems/2.2.0/gems/exif-1.0.1/.rspec
C:/Users/kart/GitHub/ruby225/lib/ruby/gems/2.2.0/gems/exif-1.0.1/.travis.yml
C:/Users/kart/GitHub/ruby225/lib/ruby/gems/2.2.0/gems/exif-1.0.1/Gemfile
C:/Users/kart/GitHub/ruby225/lib/ruby/gems/2.2.0/gems/exif-1.0.1/LICENSE.txt
C:/Users/kart/GitHub/ruby225/lib/ruby/gems/2.2.0/gems/exif-1.0.1/README.md
C:/Users/kart/GitHub/ruby225/lib/ruby/gems/2.2.0/gems/exif-1.0.1/Rakefile
C:/Users/kart/GitHub/ruby225/lib/ruby/gems/2.2.0/gems/exif-1.0.1/benchmark/benchmark.rb
C:/Users/kart/GitHub/ruby225/lib/ruby/gems/2.2.0/gems/exif-1.0.1/exif.gemspec
C:/Users/kart/GitHub/ruby225/lib/ruby/gems/2.2.0/gems/exif-1.0.1/ext/exif/data.c
C:/Users/kart/GitHub/ruby225/lib/ruby/gems/2.2.0/gems/exif-1.0.1/ext/exif/data.h
C:/Users/kart/GitHub/ruby225/lib/ruby/gems/2.2.0/gems/exif-1.0.1/ext/exif/exif.c
C:/Users/kart/GitHub/ruby225/lib/ruby/gems/2.2.0/gems/exif-1.0.1/ext/exif/exif.h
C:/Users/kart/GitHub/ruby225/lib/ruby/gems/2.2.0/gems/exif-1.0.1/ext/exif/extconf.rb
C:/Users/kart/GitHub/ruby225/lib/ruby/gems/2.2.0/gems/exif-1.0.1/lib/exif.rb
C:/Users/kart/GitHub/ruby225/lib/ruby/gems/2.2.0/gems/exif-1.0.1/lib/exif/version.rb
C:/Users/kart/GitHub/ruby225/lib/ruby/gems/2.2.0/gems/exif-1.0.1/spec/exif_spec.rb
C:/Users/kart/GitHub/ruby225/lib/ruby/gems/2.2.0/gems/exif-1.0.1/spec/sample.jpg
C:/Users/kart/GitHub/ruby225/lib/ruby/gems/2.2.0/gems/exif-1.0.1/spec/spec_helper.rb
Building native extensions.  This could take a while...
C:/Users/kart/GitHub/ruby225/bin/ruby.exe -r ./siteconf20160917-7516-k4yib7.rb extconf.rb
checking for main() in -lexif... no
checking for libexif/exif-data.h... no
creating Makefile
make "DESTDIR=" clean
make "DESTDIR="
generating exif-x64-mingw32.def
compiling data.c
data.c:1:31: fatal error: libexif/exif-data.h: No such file or directory
compilation terminated.
make: **\* [data.o] Error 1
ERROR:  Error installing exif:
        ERROR: Failed to build gem native extension.

```
Building has failed. See above output for more information on the failure.
```

make failed, exit code 2

Gem files will remain installed in C:/Users/kart/GitHub/ruby225/lib/ruby/gems/2.2.0/gems/exif-1.0.1 for inspection.
# Results logged to C:/Users/kart/GitHub/ruby225/lib/ruby/gems/2.2.0/extensions/x64-mingw32/2.2.0/exif-1.0.1/gem_make.out

Would love to use this faster gem if it can work with Windows.  Thanks!

--- a/ext/exif/extconf.rb
+++ b/ext/exif/extconf.rb
@@ -2,6 +2,7 @@
 
 require 'mkmf'
 $CFLAGS << ' -std=c99 '
+pkg_config('libexif')
 have_library('exif')
 have_header('libexif/exif-data.h')
 create_makefile('exif/exif')

